dockerfiles/cfme_tests_base/Dockerfile

```diff
@@ -1,12 +0,0 @@
-# Our custom docker image with app
```

Comment:
Yeh that's right, though the yamls need not be baked in they are pulled down, I am trying to remember why I baked them in to begin with.